src/ape/types/contract.py

```diff
@@ -39,41 +45,93 @@ def __repr__(self) -> str:
 
     @classmethod
     def from_dict(cls, params: Dict):
+        """
+        Create this class from a dictionary of its properties.
+        The dictionary keys must be the same name as the properties.
+
+        Args:
+            params (dict): A dictionary of properties.
+
+        Returns:
+            :class:`~ape.types.contract.Bytecode`
+        """
+
```

Comment:
I go back and forth about whether the dictionary methods should have a small example 🤷‍♂️

Reply:
hmm maybe we shouldn't document these method actually... I don't think users will call them too much directly